lib/pacemaker/pcmk_sched_colocation.c

```diff
@@ -172,19 +172,22 @@ cmp_primary_priority(gconstpointer a, gconstpointer b)
  *
  * \param[in,out] list        List of constraints to add \p colocation to
  * \param[in]     colocation  Colocation constraint to add to \p list
+ * \param[in]     rsc         Resource whose colocations we're getting (for
+ *                            logging only)
  *
  * \note The list will be sorted using cmp_primary_priority().
  */
 void
-pcmk__add_this_with(GList **list, const pcmk__colocation_t *colocation)
+pcmk__add_this_with(GList **list, const pcmk__colocation_t *colocation,
+                    const pe_resource_t *rsc)
 {
-    CRM_ASSERT((list != NULL) && (colocation != NULL));
+    CRM_ASSERT((list != NULL) && (colocation != NULL) && (rsc != NULL));
 
     crm_trace("Adding colocation %s (%s with %s using %s @%d) "
-              "to 'this with' list",
+              "to 'this with' list for %s",
               colocation->id, colocation->dependent->id,
               colocation->primary->id, colocation->node_attribute,
-              colocation->score);
+              colocation->score, rsc->id);
```

Comment:
pe_rsc_trace()

Reply:
Thanks, hitting it here and in the other three places